src/deadline/nuke_submitter/ui/components/scene_settings_tab.py

```diff
@@ -76,6 +76,9 @@ def _build_ui(self):
         timeouts_lyt = QGridLayout(self.timeouts_box)
         lyt.addWidget(self.timeouts_box, 5, 0, 1, -1)
 
+        self.gizmos_checkbox = QCheckBox("Include Gizmos In Job Bundle", self)
```

Comment:
Hmm... The labels for all of this _should_ be sentence-cased, but I see others that aren't. Let's not deal with that here though.

Reply:
Created https://github.com/aws-deadline/deadline-cloud-for-nuke/issues/163. Resolving